hw/ip/flash_ctrl/data/flash_ctrl.sv.tpl

```diff
@@ -377,9 +396,10 @@ module flash_ctrl import flash_ctrl_pkg::*; (
   // Program handler is consumer of prog_fifo
 
   // OTP control bits can be used to explicitly disable repair
+  // TODO, convert this to software control and use software OTP instead
```

Comment:
shall we place this bit into the HW_CFG partition?

Reply:
o yes originally i had it as part of HW_CFG, but in the end I thought it was better to leave it as a software decision.  Let me remove the "otp" wording around it, because it's really just meant to be used as a "driver" thing.

Reply:
ok got it, thanks